src/components/Todo.js

```diff
@@ -21,10 +25,17 @@ const Todo = (props) => {
   };
 
   const detailHandler = () => {
-    return <detailForm id={id} details="details" />;
```

Comment:
comonent name shall be in TitleCase @raiab

Reply:
`details="details"` will pass details as a string in that prop rather than details data to that component prop. @raiab